posts/2021-11-26-jakarta-ee-9.1.adoc

```diff
@@ -0,0 +1,347 @@
+---
+layout: post
+title: "Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12 - OpenLiberty.io
+seo-description: Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers! Several significant bug fixes are also part of this release.
+blog_description: Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers! Several significant bug fixes are also part of this release.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers! Several significant bug fixes are also part of this release.
+ 
+In link:{url-about}[Open Liberty] 21.0.0.12:
+
+* <<jee_9.1, Jakarta EE 9.1 support>>
+* <<response_headers, Configurable Response Headers>>
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A210012+label%3A%22release+bug%22[21.0.0.12].
```

Comment:
Since we already have this link in the #bugs section, lets instead change this to point to the #guides section below.  Something along the lines of 'Along with the updates to the runtime, we've also made updates to our guides'

Reply:
```suggestion
Along with the new features and functions added to the runtime, we've also made <<guides, updates to our guides>>.
```